src/components/shared/drag-upload/drag-upload.tsx

```diff
@@ -0,0 +1,64 @@
+import { ChangeEvent, forwardRef, useImperativeHandle, useRef } from "react";
+import { useDropzone } from "react-dropzone";
+import uploadIcon from "../../../assets/img/icons/global-upload.svg";
+import { TestHelper } from "../../../utils";
+import { HiddenInput, HintContainer, HintText, Wrapper } from "./drag-upload.styles";
+import { IDragUploadProps, IDragUploadRef } from "./types";
+
```

Comment:
drag-upload` handles drag & drop file and native file input interactions, it is created in the `shared` folder so future file upload components can use this component as well